yt_dlp/extractor/youtube.py

```diff
@@ -4532,7 +4533,7 @@ def process_language(container, base_url, lang_code, sub_name, query):
                     if mrr_title == 'Album':
                         info['album'] = mrr_contents_text
                     elif mrr_title == 'Artist':
-                        info['artist'] = mrr_contents_text
+                        info['artists'] = [mrr_contents_text]
```

Comment:
Can this return `[None]`?

Reply:
I think that `artists` field shouldn't be set if there is no artists.
```suggestion
                        if mrr_contents_text is not None:
                            info['artists'] = [mrr_contents_text]
```

Reply:
setting it to `None` is effectively the same as not setting it
```suggestion
                        info['artists'] = [mrr_contents_text] if mrr_contents_text else None
```